app/konnect/dev-portal/customization/index.md

```diff
@@ -6,6 +6,12 @@ content_type: how-to
 
 ## Appearance
 
+To customize Dev Portal, you have two options:
+* Basic customization using the Dev Portal [Appearance settings](#appearance) in the UI
+* Complete customization using the [open source Dev Portal client](/konnect/dev-portal/customization/self-hosted-portal/)
```

Comment:
@rosskukulinski I also added these per your feedback, do these look okay to you?

Reply:
Perfect! Thanks